tests/purs/failing/Superclasses1.out

```diff
@@ -9,6 +9,8 @@ at tests/purs/failing/Superclasses1.purs:12:1 - 13:17 (line 12, column 1 - line
 
 while checking that expression [33m#dict Su[0m
   has type [33mSu$Dict t0[0m
+while matching label [33m"Su0"[0m
+while matching label [33mcl[0m
```

Comment:
This is bad twice. The `"Su0"` hint is bad because it exposes an implementation detail of how superclasses are implemented. The `cl` hint is bad for the same reason as in 1175.